install/kubernetes/cilium/README.md

```diff
@@ -0,0 +1,313 @@
+# cilium
```

Comment:
awesome, this one looks like from https://github.com/norwoodj/helm-docs if I am not wrong. The format looks similar to me :+1: 

We can improve the template, and add Makefile target in subsequent PR :100: